src/conv.jl

```diff
@@ -24,10 +24,15 @@ padtuple(x::Tuple,p::Integer) = map(_->p, head(head(x)))
 padtuple(x::Tuple,p::Tuple) = p
 padtuple(x::AbstractArray,p) = padtuple(size(x),p)
 
-function conv(x::A, w::A; pad = 0, stride = 1, dilation = 1) where A<:AbstractArray
+function conv(x::A, w::A; pad = 0, stride = 1, dilation = 1, flipkernel=true) where A<:AbstractArray
   pad_, stride_ = padtuple(x, pad), padtuple(x, stride)
+  if flipkernel
   conv!(similar(x, cdims(size(x), dilation_dims(w, dilation), pad_, stride_)),
-        x, w, pad = pad_, stride = stride_, dilation = dilation)
+        x, w, pad = pad_, stride = stride_, dilation = dilation, flipkernel=flipkernel)
+  else
+  crosscor!(similar(x, cdims(size(x), dilation_dims(w, dilation), pad_, stride_)),
```

Comment:
It doesn't make sense to do an `if/else` here; you pass `flipkernel` through so removing it would behave the same way.

It would however be good to have an out-of-place `crosscor` function that calls `crosscor!`.